build-ps/percona-xtradb-cluster.spec

```diff
@@ -1521,9 +1521,11 @@ fi
 %doc %attr(0644,root,root) %{galera_docs}/README-MySQL
 %doc %attr(0644,root,root) %{galera_docs}/LICENSE.asio
 %doc %attr(0644,root,root) %{galera_docs}/LICENSE.crc32c
-%doc %attr(0644,root,root) %{galera_docs}/LICENSE.chromium
 %config(noreplace) %{_sysconfdir}/my.cnf
 %dir %{_sysconfdir}/my.cnf.d
+#coredumper
+%attr(755, root, root) %{_includedir}/coredumper/coredumper.h
+%attr(755, root, root) /usr/lib/libcoredumper.a
```

Comment:
please move coredumper to `%files -n percona-server-shared`

Reply:
done